src/main/kotlin/net/minecrell/pluginyml/bukkit/BukkitPlugin.kt

```diff
@@ -33,7 +33,12 @@ import org.gradle.api.artifacts.result.ResolvedComponentResult
 class BukkitPlugin : PlatformPlugin<BukkitPluginDescription>("Bukkit", "plugin.yml") {
 
     companion object {
-        @JvmStatic private val VALID_NAME = Regex("^[A-Za-z0-9 _.-]+$")
+        @JvmStatic
+        private val VALID_NAME = Regex("^[A-Za-z0-9 _.-]+$")
+        @JvmStatic
+        private val VALID_API_VERSION = Regex("^1\\.[0-9]+$")
```

Comment:
according to the [1.20.5 spigot update post](https://www.spigotmc.org/threads/spigot-bungeecord-1-20-5-1-20-6.645069/#post-4722247), api versions can now also be minor versions

Reply:
What a "nice" change. This makes it a bit more complex, but nothing too bad.